### DISCUSSION TITLE
Converting Turing Data Themes

### DISCUSSION THREAD

**Discussion by SillyNewb** [mathoudebine/turing-smart-screen-python#482] [General]
How can I convert Turing themes to python YAML? I am looking to convert this theme: HUDRNW.data

![9ca6916c-3ef9-4393-ace9-84cfc2e0b7e4](https://github.com/mathoudebine/turing-smart-screen-python/assets/21354012/34f2d2c4-19c7-4edb-9b3c-2cf9bf86e819)

Thank you in advance!

**Comment by DJRoby19** [comment]
Well rather than converting it you can play it would be simpler

**Comment by ooemir23** [comment]
wow nice theme

**Comment by mathoudebine** [comment]
Hello @SillyNewb there is no automatic way to convert Turing theme to YAML, but you can extract the images from the Turing theme then use Theme Creator to recreate it in Python
You can check this page https://github.com/mathoudebine/turing-smart-screen-python/wiki/System-monitor-%3A-themes including chapter "Import a theme from Turing / Xuanfang Windows app"

**Reply by PetroleumEngineer** [reply]
is there a way to do opposite ? i have yaml files but i want to convert them to .data

**Reply by mathoudebine** [reply]
Unfortunately the .data are a proprietary binary format made by Turing, there is no documentation of their content, so not possible to convert yaml to .data
